qa/distros/container-hosts/ubuntu_22.04.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/all/ubuntu_22.04.yaml
```

Comment:
The rook suite is referencing `qa/distros/container-hosts/ubuntu_20.04.yaml`, so it's a dangling symlink now.

Given that there doesn't seem to be any massaging needed for Ubuntu (no runc vs crun business, etc), I think it would be better to link `ubuntu_latest.yaml`, both here and from the rook suite -- just like we do for bare metal.

Reply:
> The rook suite is referencing `qa/distros/container-hosts/ubuntu_20.04.yaml`, so it's a dangling symlink now.

updated the orch/rook symlink, thanks

> Given that there doesn't seem to be any massaging needed for Ubuntu (no runc vs crun business, etc), I think it would be better to link `ubuntu_latest.yaml`, both here and from the rook suite -- just like we do for bare metal.

i'm conflicted on this part, because we often have to support two distro versions at the same time, and the use of these 'latest' symlinks ends up only testing one of them. the suites really should be using a matrix or random sampling of all supported distros

in this specific case, the version was already pinned so i just bumped the pinned version

Reply:
> in this specific case, the version was already pinned so i just bumped the pinned version

OK with me